crypto/ec/curve448/arch_32/f_impl.c

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2017-2018 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright 2014 Cryptography Research, Inc.
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ *
+ * Originally written by Mike Hamburg
+ */
+
+#include "field.h"
+
+#if (defined(__OPTIMIZE__) && !defined(__OPTIMIZE_SIZE__) && !defined(I_HATE_UNROLLED_LOOPS)) \
```

Comment:
instead of "I_HATE.." we should use "OPENSSL_SMALL_FOOTPRINT"

Reply:
Actually I deleted these and just had the unrolled loop version. I'm not sure why we would ever not want to do that.